python/remage/ipc.py

```diff
@@ -92,16 +134,20 @@ def get(self, name: str, expected_len: int = 1) -> list[str]:
         return msgs
 
     def get_single(self, name: str, default: str) -> str:
+        """Return a single value for ``name`` or ``default`` if not present."""
         gen = self.get(name)
         if len(gen) > 1:
             msg = f"ipc returned key {name} more than once"
             raise RuntimeError(msg)
         return gen[0] if len(gen) == 1 else default
 
     def set(self, name: str, values: list[str]) -> None:
+        """Replace existing entries for ``name`` with ``values``."""
```

Comment:
```suggestion
        """Replace existing stored messages for they key ``name`` with ``values``.

        .. note ::
            This will create a single-record message for each value in `values`. Setting more
            complex messages is not implemented yet.
        """
```